diff -urN aaa/banshee-0.11.0/configure.ac banshee-0.11.0/configure.ac
--- aaa/banshee-0.11.0/configure.ac	2006-09-19 04:13:39.000000000 +0200
+++ banshee-0.11.0/configure.ac	2006-09-21 22:26:47.124143624 +0200
@@ -285,8 +286,12 @@
 AC_SUBST(HAL_LIBS)
 
 dnl DBus devel detect .so version for dll maps
-PKG_CHECK_MODULES(DBUS, dbus-1 >= 0.60, dbus60=yes, dbus60=no)
-if test "x$dbus60" = "xyes"; then
+PKG_CHECK_MODULES(DBUS, dbus-1 >= 0.90, dbus90=yes, dbus90=no) 
+AC_MSG_RESULT([$dbus90])
+PKG_CHECK_MODULES(DBUS, dbus-1 >= 0.60, dbus60=yes, dbus60=no) 
+if test "x$dbus90" = "xyes"; then
+	DBUS_SOVERSION=3
+elif test "x$dbus60" = "xyes"; then
 	DBUS_SOVERSION=2
 else
 	DBUS_SOVERSION=1
@@ -294,6 +299,20 @@
 AC_SUBST(DBUS_SOVERSION)
 
 
+dnl DBus-Glib devel detect .so version for dll maps
+PKG_CHECK_MODULES(DBUS_GLIB, dbus-glib-1 >= 0.90, dbusg90=yes, dbusg90=no) 
+AC_MSG_RESULT([$dbusg90])
+PKG_CHECK_MODULES(DBUS_GLIB, dbus-glib-1 >= 0.60, dbusg60=yes, dbusg60=no) 
+if test "x$dbusg90" = "xyes"; then
+	DBUS_GLIB_SOVERSION=3
+elif test "x$dbusg60" = "xyes"; then
+	DBUS_GLIB_SOVERSION=2
+else
+	DBUS_GLIB_SOVERSION=1
+fi
+AC_SUBST(DBUS_GLIB_SOVERSION)
+
+
 dnl libnautilus-burn devel deps
 PKG_CHECK_MODULES(LIBNAUTILUS_BURN, libnautilus-burn >= 2.15, lnb15=yes, lnb15=no)
 PKG_CHECK_MODULES(LIBNAUTILUS_BURN, libnautilus-burn >= 2.13, lnb13=yes, lnb13=no)
diff -urN aaa/banshee-0.11.0/hal-sharp/hal-sharp.dll.config.in banshee-0.11.0/hal-sharp/hal-sharp.dll.config.in
--- aaa/banshee-0.11.0/hal-sharp/hal-sharp.dll.config.in	2006-07-15 17:49:38.000000000 +0200
+++ banshee-0.11.0/hal-sharp/hal-sharp.dll.config.in	2006-09-21 22:24:41.792196976 +0200
@@ -1,6 +1,6 @@
 <configuration>
   <dllmap dll="libdbus-1" target="libdbus-1.so.@DBUS_SOVERSION@" />
-  <dllmap dll="libdbus-glib-1" target="libdbus-glib-1.so.@DBUS_SOVERSION@" />
+  <dllmap dll="libdbus-glib-1" target="libdbus-glib-1.so.@DBUS_GLIB_SOVERSION@" />
   <dllmap dll="libglib-2.0.so" target="libglib-2.0.so.0" />
   <dllmap dll="libhal" target="libhal.so.1" />
 </configuration>
diff -urN aaa/banshee-0.11.0/src/banshee.exe.config.in banshee-0.11.0/src/banshee.exe.config.in
--- aaa/banshee-0.11.0/src/banshee.exe.config.in	2006-07-15 17:49:30.000000000 +0200
+++ banshee-0.11.0/src/banshee.exe.config.in	2006-09-21 22:23:33.699548632 +0200
@@ -6,4 +6,6 @@
 	<dllmap dll="gdk-x11-2.0" target="libgdk-x11-2.0.so.0" />
 	<dllmap dll="libbanshee" target="@expanded_libdir@/@PACKAGE@/libbanshee.so" />
 	<dllmap dll="libX11" target="libX11.so.6" />
+	<dllmap dll="libdbus-1" target="libdbus-1.so.@DBUS_SOVERSION@" />
+	<dllmap dll="libdbus-glib-1" target="libdbus-glib-1.so.@DBUS_GLIB_SOVERSION@" />
 </configuration>
