--- banshee-0.10.8/src/Banshee.Base/ImportManager.cs.orig	2006-03-07 20:45:33.000000000 +0100
+++ banshee-0.10.8/src/Banshee.Base/ImportManager.cs	2006-03-20 17:51:25.000000000 +0100
@@ -31,7 +31,8 @@
 using System.IO;
 using System.Threading;
 using Mono.Unix;
-using Mono.Unix.Native;
+//Ambigious Stat method between Mono.Unix and Mono.Unix.Native
+//using Mono.Unix.Native;
 using Gtk;
 
 using Banshee.Widgets;
