--- banshee-2.4.1/src/Dap/Banshee.Dap.AppleDevice/Banshee.Dap.AppleDevice/AppleDeviceTrackInfo.cs.orig	2012-06-06 21:14:55.225456207 +0200
+++ banshee-2.4.1/src/Dap/Banshee.Dap.AppleDevice/Banshee.Dap.AppleDevice/AppleDeviceTrackInfo.cs	2012-06-06 21:15:08.835382476 +0200
@@ -215,7 +215,7 @@ namespace Banshee.Dap.AppleDevice
             track.CDs = DiscCount;
             track.CDNumber = DiscNumber;
             track.TrackLength = (int) Duration.TotalMilliseconds;
-            track.Size = (int)FileSize;
+            track.Size = (uint)FileSize;
             track.Grouping = Grouping;
             try {
                 track.TimePlayed = LastPlayed;
